consensus/common/src/bls.rs

```diff
@@ -3,32 +3,38 @@ use ssz_rs::prelude::*;
 use ream_bls::BLSSignature as ReamBLSSignature;
 use ream_bls::PubKey as ReamPubKey;
 
-#[derive(Debug, SimpleSerialize, serde::Serialize, serde::Deserialize)]
+#[derive(Debug, SimpleSerialize)]
 pub struct BLSSignature {
-    pub inner: Vector<u8, 96>,
+    pub inner: [u8; 96],
```

Comment:
Curious why move away from `ssz_rs::Vector`?

I also notice but in some other places it's still being used (e.g. `DepositData`, `ExecutionPayload`, `Withdrawal`, etc.)

Reply:
Hmm, not sure about that, I forgot to change it back, will fix that